pinot-broker/src/main/java/com/linkedin/pinot/broker/routing/CfgBasedRouting.java

```diff
@@ -52,18 +49,7 @@ public void init(RoutingTableConfig cfg) {
 
   @Override
   public boolean routingTableExists(String tableName) {
-    Map<ServerInstance, SegmentIdSet> routingTableEntry = findServers(new RoutingTableLookupRequest(tableName, null, null));
-    return routingTableEntry != null && !routingTableEntry.isEmpty();
-  }
-
-  @Override
-  public void start() {
-    // Nothing to be done here
-  }
-
-  @Override
-  public void shutdown() {
-    // Nothing to be done here
+    return _cfg.getPerTableRoutingCfg().containsKey(tableName);
```

Comment:
Yes, this is an intentional change. The reason for this change is that, all other RoutingTable implementations don't check whether the routing table is empty (it will never be null if tableName exists). I changed the javadoc for the API as well.